README.md

```diff
@@ -55,7 +55,7 @@ If you're interested in contributing to OpenThread Border Router, read on.
 
 We would love for you to contribute to OpenThread Border Router and help make it even better than it is today! See our [Contributing Guidelines](https://github.com/openthread/ot-br-posix/blob/master/CONTRIBUTING.md) for more information.
 
-Contributors are required to abide by our [Code of Conduct](https://github.com/openthread/ot-br-posix/blob/master/CODE_OF_CONDUCT.md) and [Coding Conventions and Style Guide](https://github.com/openthread/ot-br-posix/blob/master/STYLE_GUIDE.md).
+Contributors are required to abide by our [Code of Conduct](https://github.com/openthread/ot-br-posix/blob/master/CODE_OF_CONDUCT.md) and [Coding Conventions and Style Guide](https://github.com/openthread/ot-br-posix/blob/master/STYLE_GUIDE.md), **except this project uses C++11**.
```

Comment:
Can we update `STYLE_GUIDE.md` rather than adding a comment here?

Reply:
updated